package.json

```diff
@@ -21,33 +21,40 @@
   "scripts": {
     "coverage": "istanbul cover _mocha -x src/model.js --  --compilers js:babel-core/register --recursive -r jsdom-global/register",
     "start": "node example/server.js",
-    "test": "mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "test": "eslint . && mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
     "build": "rimraf dist && NODE_ENV=production webpack --progress --profile --colors && NODE_ENV=production",
     "production": "npm run build && NODE_ENV=production node example/server.js"
   },
   "dependencies": {
+    "autoprefixer": "^6.4.1",
```

Comment:
I think we should override the rule in this case; less deps is definitely better. also, I think OT is inheriting the rule from AirBnB